samples/algorithms/BernsteinVazirani.qs

```diff
@@ -135,20 +135,19 @@ namespace Sample {
     operation ApplyParityOperation(
         bitStringAsInt : Int,
         xRegister : Qubit[],
-        yQubit : Qubit)
-    : Unit {
+        yQubit : Qubit
+    ) : Unit {
         // `xRegister` muts have enough qubits to represent the integer.
         let requiredBits = BitSizeI(bitStringAsInt);
         let availableQubits = Length(xRegister);
         Fact(
             availableQubits >= requiredBits,
-            $"Integer value {bitStringAsInt} requires {requiredBits} bits to " +
-            $"be represented but the quantum register only has " +
-            $"{availableQubits} qubits");
+            $"Integer value {bitStringAsInt} requires {requiredBits} bits to " + $"be represented but the quantum register only has " + $"{availableQubits} qubits"
```

Comment:
Probably best to take the `+` operators out of the string here if it's going to be all on one line anyway....

Reply:
done